components/editor/prosemirror-schema/create-new-schema.md

```diff
@@ -0,0 +1,285 @@
+                },
+            },
+
+            bullet_list: {
+                content: "list_item+",
+                group: "block",
+                attrs: { ...commonAttributes() },
+                parseDOM: [{ tag: "ul", getAttrs: getAttributes }],
+                toDOM: (node) =>
+                    hasAttrs(node.attrs) ? ["ul", getAttrs(node.attrs), hole] : ["ul", 0],
+            },
+
+            list_item: {
+                content: "block*",
+                attrs: { ...commonAttributes() },
+                parseDOM: [{ tag: "li", getAttrs: getAttributes }],
+                toDOM: (node) =>
+                    hasAttrs(node.attrs) ? ["li", getAttrs(node.attrs), hole] : ["li", 0],
+                defining: true,
+            },
+
+            // default ProseMirror table nodes
+            ...args.ProseMirror.tableNodes({
+                tableGroup: "block",
+                cellContent: "block+",
+                cellAttributes: {},
+            }),
+        };
+
+        const marks = {
+            link: {
+                attrs: {
+                    ...commonAttributes(),
+                    href: { default: null },
+                    target: { default: null },
+                    title: { default: null },
+                },
+                inclusive: false,
+                parseDOM: [{ tag: "a", getAttrs: getAttributes }],
+                toDOM(node) {
+                    return ["a", getAttrs(node.attrs), hole];
+                },
+            },
+
+            ...tagMark("strong"),
+            ...tagMark("b"),
+            ...tagMark("em"),
+            ...tagMark("i"),
+            ...tagMark("u")
+        };
+
+        const mySchema = new args.ProseMirror.Schema({ nodes, marks });
+
+        return mySchema;
+    }
+</script>
+
+<TelerikEditor @bind-Value="@TheEditorValue"
+               Schema="schemaProvider"
+               Tools="@EditorTools"
+               Width="650px"               
+               Height="400px">
+</TelerikEditor>
+
+@code {
+    string TheEditorValue { get; set; }
+
+    protected override Task OnInitializedAsync()
+    {
+        TheEditorValue = @"
+        <p>This Telerik Blazor Editor uses a ProseMirror Schema that supports only several nodes and marks:
+        <ul>
+            <li>Paragraph</li>
+            <li>Ordered and unordered lists</li>
+            <li>Hyperlinks</li>
+        </ul>
+        <p> Try editing the HTML to inserted non-supported tags such as <span>, <h1> or <img> - see how it is stripped and converted to <p>.";
+
+        return base.OnInitializedAsync();
+    }
+
+    private List<IEditorTool> EditorTools { get; set; }
+
+    protected override void OnInitialized()
+    {
+        EditorTools = new List<IEditorTool>();
+
+        EditorButtonGroup firstGroup = new EditorButtonGroup(
+            new EditorNS.Bold(),
+            new EditorNS.Italic(),
+            new EditorNS.Underline()
+        );
+        EditorTools.Add(firstGroup);
```

Comment:
These tools are irrelevant to the demo, so we can remove them. This will allow us to remove the `EditorNS` namespace alias and make the code snippet simpler and smaller.

Reply:
I consider these tools are relevant as the schema includes such marks -> https://github.com/telerik/blazor-docs/pull/2222/files/3470e12a1c942fe93341c3c1e3c35b5c0b1ca7c7#diff-ce784784fbb6b440bdc873c4bee73b840910750bfa12e39bfd06d723e6fb1360R208